packages/utils/adapter.ts

```diff
@@ -65,7 +65,7 @@ export async function runBasicTests(options: TestOptions) {
   })
 
   let user = options.fixtures?.user ?? {
-    id: randomUUID(),
+    id: id(),
```

Comment:
@balazsorban44 I made these changes in this file. We needed to specify the custom function for generating ID's (see `packages/adapter-fauna/test/index.test.ts` in this PR). But the custom function an adapter could specify was only used once in this file and not in the other cases which I changed here

Reply:
Should I open a new PR for this change?